hmpps-prisoner-search/src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/model/PrisonerModelIntTest.kt

```diff
@@ -206,11 +209,11 @@ class PrisonerModelIntTest : AbstractSearchIntegrationTest() {
       .exchange()
       .expectStatus().isOk
       .expectBody()
-      .jsonPath("allConvictedOffences").value<List<Offence>> {
-        assertThat(it).extracting("statuteCode", "offenceCode", "offenceDescription", "offenceDate", "latestBooking")
```

Comment:
this doesn't work if there are missing fields in some of the list items in the map - see https://github.com/assertj/assertj/issues/2420